tests/virt/node/migration_and_maintenance/test_multifd_policy_behavior.py

```diff
@@ -0,0 +1,36 @@
+import logging
+
+import pytest
+
+from tests.virt.node.migration_and_maintenance.utils import assert_multifd_capability_in_logs
+
+LOGGER = logging.getLogger(__name__)
+
+
+@pytest.mark.usefixtures("updated_multifd_vm_log_verbosity_config")
+class TestMultifdBehavior:
+    @pytest.mark.polarion("CNV-12266")
+    def test_multifd_disabled_with_postcopy_policy(
+        self,
+        vm_for_multifd_test,
+        migration_policy_postcopy,
+        migrated_vm_source_pod,
+    ):
+        assert_multifd_capability_in_logs(source_pod=migrated_vm_source_pod, expected_in_log=False)
```

Comment:
as per the discussion, the feature should not be tested by checking the log.

Reply:
not sure we have any other options here... only logs shows if it was really enabled or not